staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/tags/supported_resources/issubresource/doc_test.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package issubresource
 
 import (
-	"fmt"
+	fmt "fmt"
```

Comment:
```suggestion
	"fmt"
```
This is very much like the change that AI would make.

Reply:
It followed the example in zz_generated files I guess.